examples/formbot/domain.yml

```diff
@@ -0,0 +1,66 @@
+intents:
+  - greet
+  - goodbye
+  - mood_affirm
+  - mood_deny
+  - mood_great
+  - mood_unhappy
+  - request_restaurant
+
+actions:
+- utter_greet
+- utter_happy
+- utter_goodbye
+- start_restaurant
+- end_form
+- utter_explain_details_restaurant
+- utter_suggest_restaurant
+- utter_ask_cuisine
+- utter_ask_people
+- utter_ask_location
+- utter_ask_price
+- utter_book_restaurant
+- form_action
+
+form_start_actions:
```

Comment:
what do we need this for? isn't it enough that that action emits the start_form event?

Reply:
they're needed here https://github.com/RasaHQ/rasa_core/pull/878/files#diff-044625afbb4ad4ce7b830a6cd066196aR332 because during training the actions aren't accessible or carried out and message/form_actions need to be added so that the memoization policy works